conformance/ipv6.go

```diff
@@ -0,0 +1,196 @@
+		Expect(err).ToNot(HaveOccurred())
+		testNode = sriovInfos.Nodes[0]
+	})
+
+	BeforeEach(func() {
+		err := namespaces.Clean(operatorNamespace, testNamespace, clients)
+		Expect(err).ToNot(HaveOccurred())
+		Eventually(func() bool {
+			res, err := cluster.SriovStable(operatorNamespace, clients)
+			Expect(err).ToNot(HaveOccurred())
+			return res
+		}, 3*time.Minute, 1*time.Second).Should(Equal(true))
+	})
+
+	Describe("sriov", func() {
+		var _ = Context("IPv6 configured secondary interfaces on pods", func() {
+			It("should be able to ping each other", func() {
+				sriovDevice, err := sriovInfos.FindOneSriovDevice(testNode)
+				Expect(err).ToNot(HaveOccurred())
+				createSriovPolicy(testNode, sriovDevice.Name, testNode)
+				createSriovNetwork()
+			
+				pod := createTestPod([]string{"/bin/bash", "-c", "--"},
+					[]string{"while true; do sleep 300000; done;"}, testNode, ipv6NetworkName)
+				ips := getSriovNicIps(pod)
+				Expect(ips).NotTo(BeNil(), "No sriov network interface found.")
+				Expect(len(ips)).Should(Equal(1))
+				for _, ip := range ips {
+					pingPod(ip, testNode, ipv6NetworkName)
+				}
+			})
+		})
+	})
+})
+
+func createSriovNetwork() {
+	sriovNetwork := &sriovv1.SriovNetwork{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      ipv6NetworkName,
+			Namespace: operatorNamespace,
+		},
+		Spec: sriovv1.SriovNetworkSpec{
+			ResourceName:     resourceName,
+			IPAM:             `{"type": "host-local","ranges": [[{"subnet": "3ffe:ffff:0:01ff::/64"}]],"dataDir": "/run/my-orchestrator/container-ipam-state"}`,
+			NetworkNamespace: testNamespace,
+		}}
+	err := clients.Create(context.Background(), sriovNetwork)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+
+func createSriovPolicy(node string, sriovDevice string, testNode string) {
+	nodePolicy := &sriovv1.SriovNetworkNodePolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "test-policy-",
+			Namespace:    operatorNamespace,
+		},
+		Spec: sriovv1.SriovNetworkNodePolicySpec{
+			NodeSelector: map[string]string{
+				"kubernetes.io/hostname": testNode,
+			},
+			NumVfs:       numVfs,
+			ResourceName: resourceName,
+			Priority:     99,
+			NicSelector: sriovv1.SriovNetworkNicSelector{
+				PfNames: []string{sriovDevice},
+			},
+			DeviceType: "netdevice",
+		},
+	}
+
+	err := clients.Create(context.Background(), nodePolicy)
+	Expect(err).ToNot(HaveOccurred())
+
+	Eventually(func() bool {
+		stable, err := cluster.SriovStable(operatorNamespace, clients)
+		Expect(err).ToNot(HaveOccurred())
+		return stable
+	}, 5*time.Minute, 1*time.Second).Should(Equal(true))
+
+	Eventually(func() int64 {
+		testedNode, err := clients.Nodes().Get(testNode, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		resNum, _ := testedNode.Status.Capacity["openshift.io/" + resourceName]
+		capacity, _ := resNum.AsInt64()
+		return capacity
+	}, 3*time.Minute, time.Second).Should(Equal(int64(numVfs)))	
+}
+
+func getSriovNicIps(pod *k8sv1.Pod) []string {
```

Comment:
can you move this function into the utils/pod package?

I think other tests will like to use this function

Reply:
Moved to utils.network